requirements/extras/pandas_requirements.txt

```
@@ -0,0 +1 @@
+pandas==2.1.1
```

We could use [compatible releases](https://peps.python.org/pep-0440/#compatible-release) to be less strict.
```suggestion
pandas~=2.1.1
```

---

Or we could use the same version range as the one provided in
`requirements/extras/test_requirements.txt` for consistency:

https://github.com/aws/sagemaker-python-sdk/blob/4befd9320ac7768b8d64c371cf9f1bd356baebb8/requirements/extras/test_requirements.txt#L23